File: src/Nethereum.Generators.JavaScript/codeGenNodeTests/MyContractName/DTOs/TransferEventDTO.cs
using System;
using System.Threading.Tasks;
using System.Numerics;
using Nethereum.Hex.HexTypes;
using Nethereum.ABI.FunctionEncoding.Attributes;
namespace StandardToken.MyContractName.DTOs
{
    [Event("Transfer")]
    public class TransferEventDTO
    {
        [Parameter("address", "_from", 1, true )]
        public string From {get; set;}
        [Parameter("address", "_to", 2, true )]
        public string To {get; set;}
        [Parameter("uint256", "_value", 3, false )]
        public BigInteger Value {get; set;}
    }
}

File: src/Nethereum.Generators.JavaScript/codeGenNodeTestsVb/MyContractName/DTOs/TransferEventDTO.vb
Imports System
Imports System.Threading.Tasks
Imports System.Collections.Generic
Imports System.Numerics
Imports Nethereum.Hex.HexTypes
Imports Nethereum.ABI.FunctionEncoding.Attributes
Namespace StandardToken.MyContractName.DTOs

    <[Event]("Transfer")>
    Public Class TransferEventDTO
    
        <[Parameter]("address", "_from", 1, true)>
        Public Property From As String
        <[Parameter]("address", "_to", 2, true)>
        Public Property To As String
        <[Parameter]("uint256", "_value", 3, false)>
        Public Property Value As BigInteger
    
    End Class

End Namespace
